docs/serving/samples/README.md

```diff
@@ -7,9 +7,8 @@ Serving resources and how they can be applied across common use cases.
+| Cloud Events        | A quick introduction that highlights how to send and receive Cloud Events.                                                                                                                                               | [C#](./cloudevents/cloudevents-dotnet/README.md), [Go](./cloudevents/cloudevents-go/README.md), [Node.js](./cloudevents/cloudevents-nodejs/README.md), [Rust](./cloudevents/cloudevents-rust/README.md), [Java (Vert.x)](./cloudevents/cloudevents-vertx/README.md)                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | Advanced Deployment | Simple blue/green-like application deployment pattern illustrating the process of updating a live application without dropping any traffic.                                                                              |                                                                                                                                                                                                                                                                                 [YAML](./blue-green-deployment.md)                                                                                                                                                                                                                                                                                 |
-| Autoscale           | A demonstration of the autoscaling capabilities of Knative.                                                                                                                                                              |                                                                                                                                                                                                                                                                                   [Go](./autoscale-go/README.md)                                                                                                                                                                                                                                                                                   |
```

Comment:
Maybe we should update the link instead of removing it. It looks like it moved to another section.  

docs/serving/autoscaling/autoscale-go/README.md (looks like a work in progress that is being cleaned up.)

Reply:
ah here it is, thanks!